.drone.star

```diff
@@ -187,7 +187,7 @@ def build_and_test_client(ctx, c_compiler, cxx_compiler, build_type, generator,
 def gui_tests(ctx, trigger = {}, depends_on = [], filterTags = [], version = "daily-master-qa"):
     pipeline_name = "GUI-tests"
     build_dir = "build-" + pipeline_name
-    squish_parameters = "--retry 1 --reportgen stdout --reportgen json,/drone/src/test/guiTestReport"
+    squish_parameters = "--retry 1 --reportgen stdout --reportgen json,/drone/src/test/guiTestReport --envvar SQUISH_NO_CRASHHANDLER=1"
```

Comment:
Alternatively, we can set the environment variable just like we set the core dump limit in https://github.com/owncloud-ci/squish/pull/28. If you anyone has suggestion on this, then please let me know.

Reply:
I set the environment variable in squish CI before starting the squish server, so we do not need to repeat the same step here. However, I am open to any other suggestions.